### PR TITLE
Sdc 1871

### PR DESCRIPTION
**Pull request by springloops**
Hi, Streamsets Team.

I resolved [SDC-1871](https://issues.streamsets.com/browse/SDC-1871).

I added Use Custom Query configuration into JDBC Producer.
If check "Use Custom Query" then SQL Query box occur in JDBC Tab. 

If check "Use Custom Query" then Internally generates the JdbcCustomQueryRecordWriter and that execute using user's query.

I tried to create similar to the structure of the DataCollector jdbc-lib.
Could you examine my PR and merge into master branch?

I will wait for your reply.

Thank you.

### CODE REVIEW

File: jdbc-lib/src/main/java/com/streamsets/pipeline/lib/jdbc/JdbcCustomQueryRecordWriter.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
+package com.streamsets.pipeline.lib.jdbc;
```
Comment (jarcec) [inline jdbc-lib/src/main/java/com/streamsets/pipeline/lib/jdbc/JdbcCustomQueryRecordWriter.java]: Missing license header.

Comment (springloops) [inline jdbc-lib/src/main/java/com/streamsets/pipeline/lib/jdbc/JdbcCustomQueryRecordWriter.java]: Hi jarcec,
I added license header.
Thank you

**Comment by danfranks** [comment]
I would also like this functionality to be merged into the master branch.

**Comment by jarcec** [comment]
We've made several changes recently, so the patch will sadly have to be rebased. Otherwise I'm happy to finish the review and get it in :)

**Comment by jarcec** [comment]
I wonder if you're still interested in pushing the patch through @springloops ?

**Comment by jarcec** [comment]
We haven't heard back in a while, so I'll close the pull request, for now, to clean up the queue. However, if you'll get to it again, please do not hesitate and submit a new pull request @danfranks !

**Pull request closed without merging**